changelog/0.1.0.md

```diff
@@ -0,0 +1,16 @@
+# Changes in 0.1.0
+
+
```

Comment:
Do you want to explain how they are supposed to look like (such as in our GDS changelogs?

Reply:
Nah we wanted to keep it very simple for now